### ISSUE TITLE
longchat-13b-16k chat not work

### ISSUE THREAD

**Issue by ahkimkoo** [DachengLi1/LongChat#14]
reply like this:

![xxxxxxxxxx](https://github.com/DachengLi1/LongChat/assets/207163/15d36ccd-db82-40c9-8636-32731f4742f2)

**Comment by DachengLi1** [comment]
@ahkimkoo it has not been trained in Chinese data, please use only English for now.

**Comment by ahkimkoo** [comment]
> @ahkimkoo it has not been trained in Chinese data, please use only English for now.

Thank you for your reply, but even if you use English, it can't reply normally

**Comment by DachengLi1** [comment]
Can you give a screen shot on how you are loading the model, and what inputs you give?

**Comment by musabgultekin** [comment]
Because its not patched.
See here how to do that:
https://github.com/lm-sys/FastChat/blob/0a827abe0cc60a3733b4406a070beb1ac8d0e5e1/fastchat/model/model_adapter.py#L445

**Comment by scuty2000** [comment]
@DachengLi1 I would like to follow up on this. I'm having the same issue, running the same model using fastchat openai-server implementation. Getting the same outputs (some times some "A A A A A A A A A A A A" screaming) while running the latest version with the monkey patch applied.

Here are the requests I send to the endpoint and relative output:

```bash
curl http://localhost:8100/v1/chat/completions \
  -H "Content-Type: application/json" \
  -H "Authorization: Bearer fattiicazzituoi" \
  -d '{
     "model": "longchat-13b-16k",
     "messages": [{"role": "user", "content": "Say this is a test."}],
     "temperature": 0.3, "max_tokens": 200
   }'
```
`{"id":"chatcmpl-3tF6uZ7GXm54dmLwfGLQ3y","object":"chat.completion","created":1689243774,"model":"lmsys/longchat-13b-16k","choices":[{"index":0,"message":{"role":"assistant","content":"A A A A A A A A A A A A A A A A A A A A"},"finish_reason":"stop"}],"usage":{"prompt_tokens":45,"total_tokens":64,"completion_tokens":19}}`

But if I use the "completions" (non-chat) endpoint the model works "correctly" (or at least it does not scream at me):

```bash
curl http://localhost:8100/v1/completions \ 
  -H "Content-Type: application/json" \
  -H "Authorization: Bearer aaaaaaaaaaaa" \
  -d '{
    "model": "lmsys/longchat-13b-16k",
    "prompt": "Say this is a test.",
    "max_tokens": 20,
    "temperature": 0.5
  }'
```

`{"id":"cmpl-sBiuu78WYegWDnU3WDmFmF","object":"text_completion","created":1689245357,"model":"lmsys/longchat-13b-16k","choices":[{"index":0,"text":"\nYou are a test.\n\n\n\n\n\n\n\n\n\n\n\n\n\n","logprobs":null,"finish_reason":"length"}],"usage":{"prompt_tokens":7,"total_tokens":26,"completion_tokens":19}}`

TL;DR: LongChat-13B-16K goes like this:
![db637a5c-68ef-40c2-a53b-df198452bae9](https://github.com/DachengLi1/LongChat/assets/22243133/264fa9fd-f953-4632-9658-c6a220b4034f)

**Comment by DachengLi1** [comment]
@scuty2000 Fun image lol.
@merrymercy do you have an idea on this.  Is there a difference in login in completions versus chat completition (e.g. load_8_bit, patching)?

**Comment by scuty2000** [comment]
@DachengLi1 I don't know if this can help, but I suspect is related to the int8 quantization. Using the 7B version not quantized works pretty well.

**Comment by DachengLi1** [comment]
@scuty2000 Yes, I also heard it elsewhere.

**Comment by scuty2000** [comment]
Any update on this?
